packages/ketcher-core/src/application/render/raphaelRender.ts

```diff
@@ -191,11 +191,17 @@ export class Render {
     this.scrollbar.update();
   }
 
-  setMolecule(struct: Struct) {
+  setMolecule(struct: Struct, forceUpdateWithTimeout = false) {
     this.paper.clear();
     this.ctab = new ReStruct(struct, this);
     this.options.offset = new Vec2();
-    this.update(false);
+    if (forceUpdateWithTimeout) {
+      setTimeout(() => {
+        this.update(true);
+      }, 0);
+    } else {
+      this.update(false);
+    }
```

Comment:
I would propose to leave a comment why do we need this timeout to be able to easy understand it in future

Reply:
Sure, added
Thanks!